src/components/ProfileUpdate/index.tsx

```diff
@@ -151,14 +180,14 @@ const ProfileUpdate = (props) => {
           </Form.Item>
           <div className="horizontal-input">
             <Form.Item label="Year">
-              <Select
+              <CustomSelect
                 name="graduationYear"
                 value={values.graduationYear}
                 className="year"
-                onBlur={(value) => {
+                onBlur={(value: string | number) => {
                   setFieldValue('graduationYear', value);
                 }}
-                onChange={(value) => {
+                onChange={(value: string | number) => {
```

Comment:
I'm wondering why `graduationYear` here is both a string and a number. I'd assume they'd be numbers all the time.